lib/classes/class-utility.php

```diff
@@ -68,11 +68,7 @@ static private function _path_url( $instance ) {
             $path = get_template_directory_uri() . $p;
             break;
           case 'stylesheet':
-            $s = str_replace( wp_normalize_path( ABSPATH ), '', get_stylesheet_directory() );
-            $s = str_replace( '/', '\/', $s );
-            $reg = '|^(.)*(' . $s . ')(.*)$|';
-            $p = preg_replace( $reg, '$3', wp_normalize_path( dirname( __FILE__ ) ) );
-            $path = get_stylesheet_directory_uri() . $p;
+            $path = WP_CONTENT_URL . str_replace( wp_normalize_path( WP_CONTENT_DIR ), '', wp_normalize_path( dirname( __FILE__ ) ) );
```

Comment:
@alimuzzaman 

You are assuming that the ABSPATH and WP_CONTENT_DIR are related. It is a wrong assumption. WP_CONTENT_DIR can point anywhere.